crypto/fipsmodule/cpucap/cpucap.c

```diff
@@ -79,7 +79,7 @@ HIDDEN uint32_t OPENSSL_armcap_P = 0;
 
 #if defined(BORINGSSL_DISPATCH_TEST)
 // This value must be explicitly initialized to zero. See similar comment above.
-HIDDEN uint8_t BORINGSSL_function_hit[7] = {0};
+HIDDEN uint8_t BORINGSSL_function_hit[8] = {0};
```

Comment:
Does this change need to go to main irrespective of the rest of the PR?

Reply:
It already was made on main, this is cherry picking it onto this branch. https://github.com/aws/aws-lc/pull/934